src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataDeltaFeedSerializer.cs

```diff
@@ -435,10 +440,11 @@ public override async Task WriteObjectAsync(object graph, Type type, ODataMessag
         /// <param name="graph">The object to be written.</param>
         /// <param name="writer">The <see cref="ODataDeltaWriter" /> to be used for writing.</param>
         /// <param name="writeContext">The <see cref="ODataSerializerContext"/>.</param>
-        public virtual void WriteDeltaDeletedEntry(object graph, ODataWriter writer, ODataSerializerContext writeContext)
+        /// <param name="resourceContext">The <see cref="ResourceContext"/>.</param>
+        public virtual void WriteDeltaDeletedEntry(object graph, ODataWriter writer, ODataSerializerContext writeContext, ResourceContext resourceContext)
```

Comment:
>WriteDeltaDeletedEntry [](start = 28, length = 22)

since you changed the syntax of this public method, rename it as "WriteDeltaDeletedResource"